src/unittest/random_graph.cpp

```diff
@@ -0,0 +1,213 @@
+#include "random_graph.hpp"
+
+namespace vg {
+namespace unittest {
+
+VG randomGraph(int64_t seqSize, int64_t variantLen,
+                    int64_t variantCount){
+    //Create a random graph for a sequence of length seqSize
+    //variantLen is the mean length of a larger variation and variationCount
+    //is the number of variations in the graph
+
+    VG graph;
+    map<size_t, id_t> indexToNode;
+                  //Index of original sequence to node that starts at that index
+
+    string seq(seqSize, 'A');//TODO: The graph is all As
+    Node* n = graph.create_node(seq);
+    indexToNode.insert(make_pair(0, n->id()));
+
+    //Get random number generator for lengths and variation types
+    poisson_distribution<size_t> lengthDistribution(variantLen);
+    uniform_int_distribution<int> variantDistribution(0, 4);
+    uniform_int_distribution<int> indexDistribution(0, seqSize-1);
+    default_random_engine generator(time(NULL));
```

Comment:
`time(NULL)` is the standard C method for seeding the RNG, but there are better options in C++. The `random_device` is an abstract source of unpredictable bits, which is intended to be more system-independent than relying on the system clock. It acts something like a lambda function (if I recall correctly), so it would look kinda like this:
```
random_device seed_source;
default_random_engine generator(seed_source());
```